lib/views/screens/upload_screen/pages.dart

```diff
@@ -203,24 +200,24 @@ class _UploadScreenState extends State<UploadScreen> {
                       children: [
                         SizedBox(height: 10),
                         CGhostInputField(
-                          labelText: LocaleKeys.product_name,
+                          labelText: LocaleKeys.product_name.tr(),
                           onSaved: (val) => uploadItemModel.name = val,
                           validator: (value) =>
-                          value!.isEmpty ? LocaleKeys.product_name + " is required" : null,
+                          value!.isEmpty ? LocaleKeys.product_name.tr() + " is required" : null,
```

Comment:
is Required ini ganti pakai locale keys :

is_required : {} is required  
bahasa -> {} diperlukan